src/sagemaker/utils.py

```diff
@@ -1097,6 +1097,15 @@ def get_sagemaker_config_value(sagemaker_session, key):
     """
     if not sagemaker_session:
         return None
+
+    if (
+        sagemaker_session is not None
+        and sagemaker_session.settings is not None
+        and sagemaker_session.settings.ignore_sagemaker_config
+    ):
+        logger.info("Ignoring defaults config.")
```

Comment:
This should say something like the following no? (Otherwise its not clear why its being ignored)
"Ignoring defaults config because ignore_sagemaker_config was set in Session settings"

Also, now that the logging commit for the DefaultConfig is out, this should be logged to the logger from get_sagemaker_config_logger(), instead of to the sagemaker logger.